app/ts/ManualInput/ManualInputTable/CriteriaRows/CriterionRow/DataSourceRow/InputCell/EffectCell/EffectCell.tsx

```diff
@@ -20,7 +20,7 @@ export default function EffectCell({alternativeId}: {alternativeId: string}) {
   useEffect(() => {
     setLabel(
       renderInputEffect(
-        effect,
+        {...effect, unitOfMeasurementType: dataSource.unitOfMeasurement.type},
```

Comment:
since renderInputEffect is only used during manual input, it is in the wrong file. Also I think we should make an IManualInputEffect which extends the current IEffect with unitType instead of having it optional.

Reply:
Moved the function. Refactor maybe a separate story?

Reply:
Can be, if so please make a card :)